### DISCUSSION TITLE
Swipe left/right instead of up/down

### DISCUSSION THREAD

**Discussion by shuvashish76** [ReadYouApp/ReadYou#602] [Ideas]
It is nice that now we've #589 (swipe up/down) feature. I tried the debug build & noticed it has usability issues.

1. **Swipe left/right** : It's much better than up/down because suppose you're viewing an article but from a quick glance or after reading little bit of it (scroll to middle) you find it's not worth to look the whole article so better to switch to _previous_ article. Currently you've to scroll the article all the way to the _initial position_ to go to previous article which is bad for productivity. (_For next article at least we've a button for it_) Instead with swipe left/right we could switch articles from any position of the currently viewing article.
@JunkFood02 what's your opinion on it?

e.g: [NewBlur](https://f-droid.org/packages/com.newsblur/)

https://github.com/Ashinch/ReadYou/assets/48226312/00165533-e77f-4442-a0dd-8d94d6dc2d48

2. **Better transition effect** : While swipe up/down it quickly jumps to the article, would be nice if it could transition them smoothly just like NewsBlur.

3. **Bug** : The article view works weirdly if you scroll down from the 1st article or scroll up from last article. For 1st & last article these effects should be disabled.

**Comment by JunkFood02** [comment]
#589
#144
